upgrade_tests/upgrade_base.py

```diff
@@ -115,7 +119,8 @@ def prepare(self, ordered=False, create_keyspace=True, use_cache=False, use_thri
 
         if extra_config_options:
             cluster.set_configuration_options(values=extra_config_options)
-
+            
+        cluster.set_configuration_options(values={'authenticator': 'AllowAllAuthenticator', 'authorizer': 'AllowAllAuthorizer', 'role_manager': 'CassandraRoleManager'})
```

Comment:
Can't this invalidate any security upgrade tests?

Reply:
https://github.com/datastax/cassandra-dtest/pull/77/commits/ad502b52b9186ee9d1e5e63b9e97149621dff09b